.github/readme.md

```diff
@@ -1,11 +1,64 @@
-Drone is a Continuous Delivery system built on container technology. Drone uses a simple YAML configuration file, a superset of docker-compose, to define and execute Pipelines inside Docker containers. 
+# [Drone](https://www.drone.io/) <img src="https://github.com/drone/brand/blob/master/screenshots/screenshot_build_success.png" style="max-width:100px;" />
+**Welcome to the Drone codebase, we are thrilled to have you here!** 
 
-<br/>
 
-<img src="https://github.com/drone/brand/blob/master/screenshots/screenshot_build_success.png" style="max-width:100px;" />
+## What is Drone?
+Drone is a continuous delivery system built on container technology. Drone uses a simple YAML configuration file, a superset of docker-compose, to define and execute Pipelines inside Docker containers. we have two versions available: the [Enterprise Edition](https://github.com/drone/drone/blob/master/BUILDING) and the [Community Edition](https://github.com/drone/drone/blob/master/BUILDING_OSS)
+
+_Please note the official Docker images run the Drone Enterprise distribution. If you would like to run the Community Edition you can build from source by following the instructions in [BUILDING_OSS](https://github.com/drone/drone/blob/master/BUILDING_OSS)._
+ 
+## Table of Contents
+
+- [What is Drone?](#what-is-drone)
+- [Table of Contents](#table-of-contents)
+- [Community](#community)
+- [Contributing](#contributing)
+- [Code of Conduct](#code-of-conduct)
+- [Core Team](#core-team)
+- [Setup Documentation](#setup-documentation)
+- [Usage Documentation](#usage-documentation)
+- [Sample Pipeline Configuration](#sample-pipeline-configuration)
+- [Plugin Index](#plugin-index)
+
+## Community
+---
+We have a place to voice your ideas, have discussion on features, or get help with any issue you may have, please visit the community on our [Discourse](https://discourse.drone.io/) as well as our [Slack](https://join.slack.com/t/harnesscommunity/shared_invite/zt-90wb0w6u-OATJvUBkSDR3W9oYX7D~4A).
+
+
+## Contributing
+---
+We encourage you to contribute to Drone! Please check out our [Proposing Changes Guide](https://github.com/drone/proposal).
+
+## Code of Conduct
+---
+Drone follows the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+
+## Core Team
```

Comment:
many projects track this data, however, it is common practice to store in a separate file.  Perhaps we do something similar?  here are some examples:

<https://github.com/golang/go/blob/master/CONTRIBUTORS> <https://github.com/goharbor/community/blob/master/MAINTAINERS.md>

Reply:
Yeah could do something like this. Where would I put this file? We can always discuss Monday at standup.

Reply:
@bradrydzewski Could we discuss this Monday at standup so we can get this in?